go/private/tools/path.bzl

```diff
@@ -70,6 +72,13 @@ def _go_path_impl(ctx):
         for f in pkg.srcs:
             dst = pkg.dir + "/" + f.basename
             _add_manifest_entry(manifest_entries, manifest_entry_map, inputs, f, dst)
+    if ctx.attr.include_pkg:
+        for pkg in pkg_map.values():
+            for mode, f in pkg.pkgs.items():
+                # TODO(jayconrod): include other mode attributes, e.g., race.
```

Comment:
it should be possible to retain the default mode (no install suffix) for tools that don't understand it

Reply:
I'm not sure what you mean. We always need something in `pkg/<something>/path/to/lib.a`. By default, that will be `<goos>_<goarch>` for the target platform, which I think is what most tools will look for.

`installsuffix` might not actually be the right term for that string. Not sure what to call it.

Reply:
Ah, I thought you wanted to open the door for a `GOOS_GOARCH_MODE` kind of thing

Reply:
Ideally yes, but I think we should leave that as an enhancement for the future, unless you need it now.

But in that scenario, by default, I think it should match whatever go build would have put in there by default (without an explicit `-installsuffix` anywhere). So no stripped, debug, pure... but probably still `race` or `shared`.

Not sure whether that would work for everything. Need to think about it more.